poc_iot_verifier/src/runner.rs

```diff
@@ -224,8 +227,16 @@ impl Runner {
             // top level checks complete, verify the POC reports
             //
 
+            let follower =
+                follower_service::FollowerService::from_settings(&self.settings.follower)?;
```

Comment:
this is going to establish a new grpc connection to the follower node for every single PoC we process; wouldn't it be better to establish a _single_ connection to the follower here in the running and then clone the client and pass it into the `Poc::new/4` function?

Reply:
@andymck ??

Reply:
yeah that might be an issue.  I am not entirely certain of how the follower works here and the mechanics of opening and closing the connection but in theory it should be reusing the same grpc connection as the destination is the same.   In any case I will move it out into the runner